src/Sylius/Bundle/CoreBundle/Console/Command/InstallSampleDataCommand.php

```diff
@@ -13,21 +13,34 @@
 
 namespace Sylius\Bundle\CoreBundle\Console\Command;
 
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Bundle\CoreBundle\Installer\Checker\CommandDirectoryChecker;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Helper\QuestionHelper;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ConfirmationQuestion;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
+#[AsCommand(
+    name: 'sylius:install:sample-data',
+    description: 'Install sample data into Sylius.',
+)]
 final class InstallSampleDataCommand extends AbstractInstallCommand
 {
-    protected static $defaultName = 'sylius:install:sample-data';
+    public function __construct(
+        protected readonly EntityManagerInterface $entityManager,
+        protected readonly CommandDirectoryChecker $commandDirectoryChecker,
+        protected readonly bool $publicDirectoryPath,
```

Comment:
could be "publicDir" to be closer to "kernel.project_dir" and "kernel.cache_dir" naming.